### ISSUE TITLE
module 'torch.nn.functional' has no attribute 'Sigmoid'

### ISSUE THREAD

**Issue by Jimut123** [bigmb/Unet-Segmentation-Pytorch-Nest-of-Unets#45]
Hi Malav,

While I was using  AttU_Net and R2ATTU_Net, it throws some error, could you help regarding this issue...

I think you might have noticed this issue, but have missed somehow [in this file](https://github.com/bigmb/Unet-Segmentation-Pytorch-Nest-of-Unets/blob/master/pytorch_run.py).

```
#passsing this string so that if it's AttU_Net or R2ATTU_Net it doesn't throw an error at torchSummary

>> Might be something missing here..

model_test = model_unet(model_Inputs[0], 3, 1)

model_test.to(device)
```

Best,
-Jimut

**Comment by Jimut123** [comment]
Ahh, my bad, there was some modifications done by me later! Now it works...
